packages/cloud-cognitive/src/components/NonLinearReading/NonLinearReading.stories.js

```diff
@@ -17,40 +18,153 @@ import mdx from './NonLinearReading.mdx';
-// just a pill for a term, expanding to show its definition.
+// just a pill for a keyword, expanding to show its definition.
 // Should always be shown in context with surrounding text.
-const Template = (args) => {
+
+const TemplateSingleLevel = (args) => {
+  const theme = getSelectedCarbonTheme();
+
   return (
-    <>
+    <div className={`${storyClass}__viewport`}>
       XDR Connect’s correlation
-      <NonLinearReading {...args} />
+      <NonLinearReading {...args} theme={theme} />
       creates a case by processing data from alerts across multiple security
       tools.
-    </>
+    </div>
+  );
+};
+
+const TemplateMultipleLevel = (args) => {
+  const theme = getSelectedCarbonTheme();
+
+  return (
+    <div className={`${storyClass}__viewport`}>
+      Findings are created by the alerts{' '}
+      <NonLinearReading
+        {...args}
+        definition={
+          <>
+            We examine the alerts from each source, and{' '}
+            <NonLinearReading
+              {...args}
+              definition="Correlation allows us to identify connections between common
+                  observables, tactics, and techniques, and to remove any
+                  duplicate data, thereby streamlining the investigation for
+                  you."
+              theme={theme}
+            >
+              correlate
+            </NonLinearReading>{' '}
+            them together with{' '}
+            <a href="https://www.ibm.com/" target="_blank" rel="noreferrer">
+              rules
+            </a>
+            . While a case can contain multiple alerts, a single finding can
+            only be created from a single alert. So, when you select a finding,
+            you can drill right down to the raw data behind it: the payload. We
+            also use our
+            <NonLinearReading
+              {...args}
+              definition="
+                  Our threat intelligence service contains an enrichment
+                  capability. During the enrichment process, we add context that
+                  is used to establish the severity of artifacts associated with
+                  a finding. This is what determines the severity of the finding
+                  itself. Bear in mind that each case can have multiple
+                  findings, and every finding will have its own severity.
+                "
+              theme={theme}
+            >
+              threat intelligence service
+            </NonLinearReading>
+            to establish the severity of the artifacts.
+          </>
+        }
+        theme={theme}
+      >
+        ingested
+      </NonLinearReading>{' '}
+      from your own security systems. The findings here are confirmed findings
+      that have been created from alerts ingested from your own sources, before
+      being enriched to create cases.
+    </div>
   );
 };
 
-export const Default = prepareStory(Template, {
+const TemplateWithGradientBG = (args) => {
```

Comment:
We typically write things out story names instead of using abbreviations.

```suggestion
const TemplateWithGradientBackground = (args) => {
```

Reply:
Done